pkg/prometheus/prometheus.go

```diff
@@ -62,6 +62,14 @@ var (
 		[]string{"client", "code", "method"},
 	)
 
+	clientErrorCounter = prometheus.NewCounterVec(
```

Comment:
I think `clientErrorCounter` needs to be registered in the `init()` function in order to be accounted for.